### PR TITLE
Subsystems Complete

### PR DESCRIPTION
**Pull request by vladaviedov**
Subsystems + manual commands have been written and tested

### CODE REVIEW

File: src/main/java/org/recordrobotics/munchkin/Constants.java [hunk 1]
```diff
@@ -13,6 +13,8 @@
  * constants are needed, to reduce verbosity.
  */
 /**
- * TODO: PROGRAMMER NOTE: organize constants alphabetically, first by file name then by variable name
+ * TODO: PROGRAMMER NOTE: organize constants alphabetically by variable name
  */
-public final class Constants {}
+public final class Constants {
+	// maximum limit for motors to spin, used in all subsystems
```
Comment (fielia) [inline src/main/java/org/recordrobotics/munchkin/Constants.java:19]: welp
what do we do about this comment

**Review by fielia — APPROVED** [review]
honestly think it's fine

File: src/main/java/org/recordrobotics/munchkin/Constants.java (hunk 1 above)
Comment (fielia) [reply src/main/java/org/recordrobotics/munchkin/Constants.java:19]: like, it's not supposed to be there (it describes a variable that is no longer there)

Comment (vladaviedov) [reply src/main/java/org/recordrobotics/munchkin/Constants.java:19]: ah i see

**Review by fielia — APPROVED** [review]

**Review by DragonSM — APPROVED** [review]
Approving for now but someone needs to remember to fix the limit switch port in acquisition's naming from "LIMIT_SWITCH" to "LIMIT_SWITCH_PORT" at some point

**Pull request merged**